.gitignore

```diff
@@ -7,6 +7,7 @@
 /_src/img/
 /_src/website-redesign/
 .vscode/
+_gen/
```

Comment:
Hugo's docs recommend to check them in because these resource take time to build, and the server or users will have to build all images of the website from scratch on the first build. For scss it's fast, but image processing is not. And I'd like us to use hugo's processing automatically about everywhere on the website moving forward, so I wouldn't ignore these files.

Reply:
Alright, I didn't find any info on this, but I reverted .gitignore changes.